test/query.test.js

```diff
@@ -225,6 +225,17 @@ describe('query', () => {
           api_version: '3',
         })
       ),
+      assertQuery(
+        query.Let(
```

Comment:
@marrony @BrunoQuaresma Changing the order of the fields in this object causes the test to fail. However, I don't think we've necessarily introduced this as a bug 🤔 Thoughts?

Reply:
This test looks good :D